packages/php/email-editor/composer.json

```diff
@@ -16,7 +16,8 @@
     ]
   },
   "require": {
-    "php": ">=7.4"
+    "php": ">=7.4",
+    "soundasleep/html2text": "^2.1"
```

Comment:
You added a new dependency, but I am not sure if we install those dependencies during the build. What I have tested, after deleting installed dependencies from this PHP package, it was not installed during the build preparation. Please, can you check that?

Reply:
I am sorry, but I am unsure what you are referring to. Could you please point me to the code that does the build and where I should orchestrate the deps installation?

Reply:
I think you should focus on the command `./do install` and the file `build.sh`. Those are places where I would expect changes.